server/src/test/java/org/elasticsearch/indices/flush/FlushIT.java

```diff
@@ -369,4 +382,29 @@ public void testDoNotRenewSyncedFlushWhenAllSealed() throws Exception {
         assertThat(forthSeal.successfulShards(), equalTo(numberOfReplicas + 1));
         assertThat(forthSeal.syncId(), not(equalTo(thirdSeal.syncId())));
     }
+
+    public void testFlushOnInactive() throws Exception {
+        final String indexName = "flush_on_inactive";
+        List<String> dataNodes = internalCluster().startDataOnlyNodes(2, Settings.builder()
+            .put(IndexingMemoryController.SHARD_INACTIVE_TIME_SETTING.getKey(), randomTimeValue(2, 5, "s")).build());
```

Comment:
Could we maybe make the setting smaller to ensure the test completes faster? Maybe use:

```
TimeValue.timeValueMillis(scaledRandomIntBetween(10, 2000))
```
instead?

Reply:
Adjusted in https://github.com/elastic/elasticsearch/pull/49126/commits/330c108b1038d2f18516fb4014d2068d29bf3d09.